lib/galaxy/datatypes/msa.py

```diff
@@ -11,6 +11,87 @@
 log = logging.getLogger(__name__)
 
 
+class Infernal_CM_1_1(Text):
```

Comment:
Let's call this class InfernalCM and add a MetadataElement(name="version", ...) where we store the Infernal CM version. This way we just need one class for future versions of this file format.

Reply:
Looks fine? Should it be optional?
` MetadataElement(name="cm_version", default=1.1, desc="CM Infernal version ",  readonly=True, visible=True, optional=True, no_value=0)`

Reply:
desc could be "Infernal Covariance Model version" and name simply "version"?
You can later access this information via `$file.metadata.version`

Reply:
CM versions might not be compatible. How expose the version info to the user?